mx.py

```diff
@@ -7670,6 +7670,8 @@ def prepare(self, sourceFiles, project, outputDir, classPath, processorPath, sou
             javacArgs += ['-proc:none']
         c = str(compliance)
         javacArgs += ['-target', c, '-source', c]
+        if c == '19' and self.jdk.javaCompliance == '19':
```

Comment:
I think this should be `>=` instead of `==`. What happens when we get to Java 20 and loom is still in preview?

Reply:
This PR is not ready for review, and this is a temporary solution for local tests. Besides, I think the preview feature in JDK is very tricky for Graal, E.g. `jdk.internal.access.foreign.MemorySegmentProxy` has been moved to a preview feature in JDK-8282191, which affects some substitutions in `Target_java_nio_DirectByteBuffer_JDK17OrLater.java`.